tooling/template-base/components/LearnerAccess/Views/DisplayListView.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import dashboardDefault from '../../../renderer/dashboardDefault.png';
+import dropDownClosed from '../../../renderer/dropDownClosed.svg';
+import dropDownOpen from '../../../renderer/dropDownOpen.svg';
+import LearnerAccessListDisplayDropDown from './ListDisplayDropDown';
+import {
+  HydratedContentItem,
+  useUserCourseCompletionProgressQuery
+} from '@thoughtindustries/content';
+import clsx from 'clsx';
+
+interface ContentUiProps {
+  item: HydratedContentItem;
+  index?: number;
+}
+
+const LearnerAccessDisplayListView = ({ item }: ContentUiProps) => {
+  const [listViewDropDown, setListViewDropDown] = useState(false);
+
+  const { data } = useUserCourseCompletionProgressQuery({
+    variables: {
+      id: item.id
+    },
+    fetchPolicy: 'network-only'
+  });
+
+  const DropDownClassNames = listViewDropDown ? 'border-l-4 border-blue-700' : '';
+
+  return (
+    <>
+      <div className={clsx(['flex flex-col w-full', DropDownClassNames])}>
```

Comment:
Since you're not using `DropDownClassNames` anywhere else, think this could just be


```javascript
clsx('flex flex-col w-full', listViewDropDown && 'border-1-4 border-blue-700')
```